### PR TITLE
Configuration Module Test improvements (and bug fixes)

### PR DESCRIPTION
**Pull request by rossriley**
This takes tests/Configuration up to 100% code coverage
Plus:
Fixes a bug in LowlevelException where the file search path was wrong.

### CODE REVIEW

**Comment by bobdenotter** [comment]
f yeah

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,7 @@ files/*
 extensions/*
 thumbs/
 vendor/
+phpunit.xml
```
Comment (GawainLynch) [inline .gitignore]: You beat me to it

**Comment by GawainLynch** [comment]
Something weird here:
```
$ phpunit
Bolt - Fatal Error
Postgres was selected as the database type, but the driver does not exist or is not loaded. Please install the pdo_pgsql driver.
```

File: tests/Configuration/LowlevelChecksTest.php [hunk 2]
```diff
@@ -84,6 +89,24 @@ public function testPlatformFailsMysql()
...
     
+    public function testGoodMysql()
+    {
+        $check = $this->getMockedChecker('mockPostgres'); 
+        $check->doDatabaseCheck(); 
+    }
+    
+    public function testGoodPostgres()
+    {
+        $check = $this->getMockedChecker('mockMysql');
```
Comment (GawainLynch) [inline tests/Configuration/LowlevelChecksTest.php]: Ah, I see. Line 94 and 100 are the wrong way around.

**Pull request merged**